src/Cameras/babylon.camera.ts

```diff
@@ -413,8 +414,13 @@
             return this._computedViewMatrix;
         }
 
+        public setForcedProjectionMatrix(array: number[]): void {

```

Comment:
Done. But I did it directly in github (first time I noticed we can edit files here ;-) ), so I didn't tried to compile TS file (but it should pass, the change is so tiny I couldn't barely have made a typo)